packages/bruno-app/src/components/Sidebar/TitleBar/StyledWrapper.js

```diff
@@ -1,6 +1,160 @@
 import styled from 'styled-components';
 
 const StyledWrapper = styled.div`
+  .titlebar-container {
+    display: flex;
+    align-items: center;
+  }
+
+  .workspace-icon-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .workspace-letter-logo {
```

Comment:
Classes `workspace-letter-logo` `workspace-icon-container` are not used. Please remove them